src/app/components/collectibles/src20-image.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+
+import { css } from 'leather-styles/css';
+
+import { Eye1ClosedIcon } from '@leather.io/ui';
+
+interface Src20ImageProps {
+  alt?: string;
+  src: string;
+}
+export function Src20Image(props: Src20ImageProps) {
+  const { alt, src } = props;
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+  const [width, setWidth] = useState(0);
+
+  if (isError) return <Eye1ClosedIcon />;
```

Comment:
Not sure here, maybe just do the `Src20AvatarIcon` instead?

Reply:
Yeah, that's a good call.

Reply:
Done